File: demo-vk-rest-scraping.java/src/main/java/demo/model/ConstructorMediator.java
package demo.model;

import com.google.api.client.util.ArrayMap;

public class ConstructorMediator {

    private final int id;
    private final String title;

    public ConstructorMediator(ArrayMap am) {
        this.id = Integer.valueOf(am.get("id").toString());
        this.title = am.get("title").toString();
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/ConstructorMediator.kt
package demo.model

import com.google.api.client.util.ArrayMap

class ConstructorMediator(am: ArrayMap<*, *>) {

    val id: Int
    val title: String

    init {
        this.id = Integer.valueOf(am["id"].toString())!!
        this.title = am["title"].toString()
    }
}
